wpilibcIntegrationTests/src/MotorInvertingTest.cpp

```diff
@@ -18,20 +18,21 @@ using namespace frc;
 enum MotorInvertingTestType { TEST_VICTOR, TEST_JAGUAR, TEST_TALON };
 static const double motorSpeed = 0.15;
 static const double delayTime = 0.5;
-std::ostream& operator<<(std::ostream& os, MotorInvertingTestType const& type) {
+std::ostream& operator<<(std::ostream& oss,
+                         MotorInvertingTestType const& type) {
```

Comment:
These are some of the variable renames I mentioned in the commit message. We currently mix "os" and "oss", so I chose the latter.